app/components/preprint-status-banner/style.scss

```diff
@@ -127,6 +127,10 @@
             margin: 10px 10px 5px 10px;
           }
         }
+
+        .comment-length-error {
+          color: red;
```

Comment:
Prefer using a variable in `app/styles/_variables.scss` for colors.

Reply:
`$state-danger-text` is the color that the preprint form uses for validation errors.